pkgs/development/python-modules/pyvista/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, imageio
+, numpy
+, pillow
+, pooch
+, scooby
+, vtk
+}:
+
+buildPythonPackage rec {
+  pname = "pyvista";
+  version = "0.37.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-02osbV9T9HOrapJBZpaTrO56UXk5Tcl1ldoUzB3iMUE=";
+  };
+
+  # Disable vtk detection as in mayavi
+  postPatch = ''
+    sed -i '/vtk/d' setup.py
+  '';
+
+  propagatedBuildInputs = [
+    imageio
+    numpy
+    pillow
+    pooch
+    scooby
+    vtk
+  ];
+
+  # Tests require pgrep
```

Comment:
pgrep from pcrocps does not work?

Reply:
I think the comment is wrong, so I removed it. I guess I'll test with `pytestCheckHook`, but I don't have enough computing resources right now.